packages/relay/src/lib/eth.ts

```diff
@@ -1905,22 +1905,8 @@ export class EthImpl implements Eth {
   }
 
   private async getAccountLatestEthereumNonce(address: string, requestId?: string) {
-    const cachedResponse: any = this.mirrorNodeClient.getIsValidContractCache(address);
-
-    // address is a valid contract
-    if (cachedResponse) {
-      return EthImpl.oneHex;
-    }
-
     const accountData = await this.mirrorNodeClient.getAccount(address, requestId);
     if (accountData) {
-      if (!accountData.ethereum_nonce) {  // if nonce > 0 then the entity cannot be a contract
-        const contract = await this.mirrorNodeClient.isValidContract(address, requestId, 0);
-        if (contract) {
-          return EthImpl.oneHex;
-        }
-      }
-
       return EthImpl.numberTo0x(accountData.ethereum_nonce);
```

Comment:
Actually we now have to handle the null case.
Historical contracts that didn't track the nonce will return a null on this.
In this case you can be assured it's a contract and you should send back `EthImpl.oneHex`
e.g. https://previewnet.mirrornode.hedera.com/api/v1/accounts/0.0.1036

Reply:
```suggestion
      // with HIP 729 ethereum_nonce should always be 0+ and null. Historical contracts may have a null value as the nonce was not tracked, return default EVM compliant 0x1 in this case
      return accountData.ethereum_nonce ? EthImpl.numberTo0x(accountData.ethereum_nonce) : EthImpl.oneHex;
```